### ISSUE TITLE
Flash device

### ISSUE THREAD

**Issue by carlosalvarez91** [mozmorris/react-webcam#189]
Is there any way to use the camera flash on capture?
Thanks

**Comment by mozmorris** [comment]
@carlosalvarez91 thank you for posting this question. It's a good question since this is actually possible on _some_ devices with the right Browser installed. Support right now is limited to Chrome and Opera on the devices that do support it.

See: https://developer.mozilla.org/en-US/docs/Web/API/PhotoCapabilities/fillLightMode for more information. 

I will leave this issue open, as this is something we would like to support.

**Comment by studioswong** [comment]
Thank you for the follow up @mozmorris ! Wondering before the support for accessing camera flash is released for react-webcam, what is your suggested work-around in the meantime to allow my web-app to access the camera flash settings 
while still using react-webcam to control our camera? Having built our app around react-webcam, we LOVE react-webcam and would hope to continue working with react-webcam with a temporary work-around while the support for flash is being worked on. 

Thank you so much btw, keep up the great work!

**Comment by will-weiss** [comment]
Yes, thanks @mozmorris for an awesome project. Following up in the same situation @studioswong. Thanks so much for your attention and your work on this!

**Comment by braveheart84** [comment]
Joining in to say how awesome this project is!

And also would love to see how we can incorporate the flash light together with react-webcam

**Comment by Nikkolas-Cage** [comment]
bump <3 any update with this?

**Comment by alexrqs** [comment]
@mozmorris what if the solution is to use same as front camera for mobile and paint white or yellow pixels from the screen (in this case from the camera area with some sort of fast transition), and could even do 3 flash attempts to reduce red eyes effect, what do you think?

**Comment by harshada1810** [comment]
@mozmorris  Excellent project! (y) Any update on this?
